### PR TITLE
feat(collapsible/basic): add border variable.

### PR DESCRIPTION
**Pull request by betobe**

### CODE REVIEW

File: components/collapsible/basic/src/index.scss [hunk 1]
```diff
@@ -1,8 +1,12 @@
 @import '~@schibstedspain/theme-basic/lib/index';
 
+$bdc-collapsible-basic: $c-gray-lightest !default;
+$bd-collapsible-basic: 1px solid $bdc-collapsible-basic !default;
+
 .sui-CollapsibleBasic {
   background-color: $bgc-collapsible-basic;
   width: 100%;
+  border: $bd-collapsible-basic;
```
Comment (ronaldo-mh) [inline components/collapsible/basic/src/index.scss]: Did you check this with sass lint? The border property should be below background-color.

**Review by ronaldo-mh — APPROVED** [review]

File: components/collapsible/basic/src/index.scss (hunk 1 above)
Comment (betobe) [reply components/collapsible/basic/src/index.scss]: 👍

File: components/collapsible/basic/src/index.scss [hunk 2]
```diff
@@ -1,8 +1,12 @@
 @import '~@schibstedspain/theme-basic/lib/index';
 
+$bdc-collapsible-basic: $c-gray-lightest !default;
+$bd-collapsible-basic: 1px solid $bdc-collapsible-basic !default;
```
Comment (JoanClaret) [inline components/collapsible/basic/src/index.scss]: Are those variables already defined in `theme-basic`?

Comment (betobe) [reply components/collapsible/basic/src/index.scss]: 👍  Now they are. Removing them from here soon.

**Review by JoanClaret — APPROVED** [review]

File: components/collapsible/basic/src/index.scss (hunk 2 above)
Comment (JoanClaret) [reply components/collapsible/basic/src/index.scss]: cool!

**Pull request merged**